### PR TITLE
Add toggle to hide OpponentInfo targets target

### LINKED ISSUE #4361: Toggle for OpponentsOpponentName
Would like a toggle showing the player being targeted by the NPC. Even though it only shows in multi, there's no need for it for solo players which most pvm is nowdays or people don't find it useful. Differing sizes of the box also makes it awkward when wanting the opponent health box in a specific position around other plugin boxes (box enlargens to fit room for player name in multi from standard size when in singles). Owner of a previous thread requesting this closed his own thread but a label tag was added. https://github.com/runelite/runelite/issues/2570

### PR DESCRIPTION
**Pull request by forsco**
Implements a config toggle to hide the name of your Targets target.

Closes #4361

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java [hunk 1]
```diff
@@ -52,4 +52,15 @@ default boolean showPercent()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "hideTargetsTarget",
+		name = "Hide targets target",
+		description = "Configures whether your targets target will be visible",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java]: `target's`

File: runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java [hunk 2]
```diff
@@ -52,4 +52,15 @@ default boolean showPercent()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "hideTargetsTarget",
+		name = "Hide targets target",
```
[the remark this answers is not in the document]
Comment (forsco) [reply runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java]: Fixed :)

**Comment by forsco** [comment]
This is pending a merge, approved for almost a month now.

**Comment by Adam-** [comment]
Took #6326 instead

**Comment by MagicfTail** [comment]
Isn't this a duplicate of #4598 ?

**Pull request closed without merging**